pkg/restclient/device.go

```diff
@@ -15,3 +17,22 @@ func (c *Client) DeviceAdd(peerid string, device string) (deviceapi.AddDeviceRes
 	err := c.post("/v1/devices/"+peerid, req, http.StatusOK, &peerinfo)
 	return peerinfo, err
 }
+
+// DeviceList lists the devices
+func (c *Client) DeviceList(peerid string) ([]deviceapi.Info, error) {
+	var deviceList deviceapi.ListDeviceResp
+	url := fmt.Sprintf("/v1/devices/%s", peerid)
+	err := c.get(url, nil, http.StatusOK, &deviceList)
+	return deviceList, err
+}
+
+// DeviceEdit edits devices
+func (c *Client) DeviceEdit(peerid string, device string, state string) error {
```

Comment:
`(peerid string, device string, state string)`  to `(peerid, device, state string)`